src/main/java/org/candlepin/guice/CustomizableModules.java

```diff
@@ -51,6 +51,10 @@ public Set<Module> customizedConfiguration(Map<String, String> moduleConfig) {
             Set<Module> toReturn = new HashSet<>();
 
             for (Map.Entry<String, String> entry : moduleConfig.entrySet()) {
+                if (entry.getValue() == null || entry.getValue().isEmpty()) {
+                    continue;
+                }
```

Comment:
I ran into an issue where the `entry.getValue()` will be an empty string here if the environment variable (MODULE_CONFIG_HOSTED_CONFIGURATION_MODULE) is not set to anything. This occurs for our standalone tests where we don't want need to set this value to the AdapterOverrideModule.